lib/SPIRV/SPIRVLowerConstExpr.cpp

```diff
@@ -132,18 +132,18 @@ SPIRVLowerConstExpr::visit(Module *M) {
           if (auto CE = dyn_cast<ConstantExpr>(Op)) {
             SPIRVDBG(dbgs() << "[lowerConstantExpressions] " << *CE;)
             auto ReplInst = CE->getAsInstruction();
-            ReplInst->insertBefore(FBegin->begin());
```

Comment:
Can we insert it at the end of the first BB? Then redundant instructions are easier to be merged by optimizations.